File: Euclid/Messages/Outgoing/LOGOUT.cs
using Euclid.Network.Streams.Util;

namespace Euclid.Messages.Outgoing
{
    class LOGOUT : IMessageComposer
    {
        private string name;

        public LOGOUT(string name)
        {
            this.name = name;
        }

        public override void Write()
        {
            Data.Add(new ArgumentEntry(this.name));
        }
    }
}

File: euclid/src/main/java/euclid/messages/outgoing/LOGOUT.java
package euclid.messages.outgoing;

import euclid.messages.IMessageComposer;
import euclid.network.streams.util.ArgumentEntry;

public class LOGOUT extends IMessageComposer {

    private final String name;

    public LOGOUT(String name) {
        this.name = name;
    }

    @Override
    public void write() {
        getData().add(new ArgumentEntry(name));
    }
}
